boefjes/boefjes/storage/memory.py

```diff
@@ -134,6 +134,20 @@ def get_enabled_normalizers(self, organisation_id: str) -> list[str]:
             if enabled and "norm" in plugin_id
         ]
 
+    def get_disabled_boefjes(self, organisation_id: str) -> list[str]:
+        return [
+            plugin_id
+            for plugin_id, enabled in self._enabled.get(organisation_id, {}).items()
+            if not enabled and "norm" not in plugin_id
+        ]
+
+    def get_disabled_normalizers(self, organisation_id: str) -> list[str]:
+        return [
+            plugin_id
+            for plugin_id, enabled in self._enabled.get(organisation_id, {}).items()
+            if not enabled and "norm" in plugin_id
+        ]
```

Comment:
I think checking for normalizers like this isn't reliable. For example, see the list produced by running:
`find boefjes/boefjes/plugins -name normalizer.json -execdir grep id {} \;`

You would see that most normalizer IDs end with `_normalize`. But there are exceptions like `kat_report_data`, `kat_answer_parser`, `adr-validator-normalize`, etc. With the current approach it's also possible that a boefje ID could contain the string 'norm' even if it's unrelated. I think we need to find a more explicit or consistent way to identify normalizers?